internal/action/command.go

```diff
@@ -479,7 +479,10 @@ func SetGlobalOptionNative(option string, nativeValue interface{}) error {
 		}
 	}
 
-	if !local {
+	if local {
+		MainTab().CurPane().Buf.SetOptionNative(option, nativeValue)
+		return nil
+	} else {
```

Comment:
`else` is not needed after `return`, and that's what I actually had in mind. We can move the bulk of the function out of this `if`, since we are done with the `local` case and can forget about it.

For cleanliness all this can be done in 2 separate commits (no matter in which order): the actual fix for unneeded rewriting and the refactoring.